.babelrc

```diff
@@ -1,30 +1,3 @@
 {
-  "presets": [
-    [
-      "env",
-      {
-        "useBuiltIns": true,
-        "targets": {
-          "node": "4.8"
-        },
-        "exclude": [
-          "transform-async-to-generator",
-          "transform-regenerator"
-        ]
-      }
-    ]
-  ],
-  "plugins": [
-    "transform-runtime"
-  ],
-  "env": {
-    "test": {
-      "presets": [
-        "env"
-      ],
-      "plugins": [
-        "transform-runtime"
-      ]
-    }
-  }
+  "presets": ["@babel/preset-env"]
```

Comment:
Does `preset-env` read the `target` from `pkg.engines` now? Minimium is `node >= v6.0.0` in case you need it